kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializer.java

```diff
@@ -242,18 +243,26 @@ private void handleSessionSerialization(String sessionId,
                     return;
                 }
             }
+        } catch (NotSerializableException e) {
+            getLogger().trace(
+                    "Optimistic serialization of session {} with distributed key {} failed,"
+                            + " some attribute is not serializable. Giving up immediately since the error is not recoverable",
+                    sessionId, clusterKey, e);
```

Comment:
Should this be at `warn` or `error` level to be more visible?

Reply:
Good catch

Reply:
done